app.js

```diff
@@ -0,0 +1,192 @@
+              game.currentPlay.symbol === "X" ? game.oPlayer : game.xPlayer;
+            document.getElementById("player").innerHTML = game.currentPlay.name;
+
+            if (
+              board[0].innerHTML === board[1].innerHTML &&
+              board[1].innerHTML === board[2].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[3].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[5].innerHTML &&
+              board[3].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[6].innerHTML === board[7].innerHTML &&
+              board[7].innerHTML === board[8].innerHTML &&
+              board[6].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[0].innerHTML === board[3].innerHTML &&
+              board[3].innerHTML === board[6].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[1].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[7].innerHTML &&
+              board[1].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[2].innerHTML === board[5].innerHTML &&
+              board[5].innerHTML === board[8].innerHTML &&
+              board[2].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[0].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[8].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[2].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[6].innerHTML &&
+              board[2].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else {
+              game.playCount++;
+              if (game.playCount === 9) {
+                document.getElementById("reset").classList = "";
+              }
+            }
+          }
+        });
+      }
+    }
+  };
+
+  const showWinner = function (game) {
+    document.getElementById("winner").innerHTML =
+      game.currentPlay.name === game.oPlayer.name
+        ? game.xPlayer.name
+        : game.oPlayer.name;
+
+    document.getElementById("message").style.display = "block";
+    document.getElementById("game-status").innerHTML = "status = game over";
+    game.status = "Game Over";
+    document.getElementById("reset").classList = "";
+  };
+
+  return {
+    showBoard: function (game) {
+      boardWinner(game);
+    },
+
+    newGame: function (xPlayer, oPlayer) {
+      return new GameBoard(xPlayer, oPlayer);
+    },
+  };
+})();
```

Comment:
[Optional]

Some feedback that should be added/removed
- When there's a winner, let's remove the information that the next player should make their move (`Now your turn !`)
   ![image](https://user-images.githubusercontent.com/36057474/82726600-38086080-9cdd-11ea-8f00-7e9f1d94a559.png)
- This is the same for `draw`, let's hide this information and announce that the game is a draw
   ![image](https://user-images.githubusercontent.com/36057474/82726624-60905a80-9cdd-11ea-88d9-3c905bfb7183.png)